src/TTrigger.cpp

```diff
@@ -570,7 +570,9 @@ inline void TTrigger::updateMultistates(int regexNumber, std::list<std::string>&
         mConditionMap[pCondition] = pCondition;
         pCondition->multiCaptureList.push_back(captureList);
         pCondition->multiCapturePosList.push_back(posList);
-        pCondition->nameCaptures.push_back(*nameMatches);
+        if (nameMatches != nullptr) {
```

Comment:
:information_source: the ` != nullptr` is (unnecessarily) verbose IMHO. The more concise form would be:
```diff
-        if (nameMatches != nullptr) {
+        if (nameMatches) {
```